src/client/components/features/ConsulterOffre.module.scss

```diff
@@ -88,7 +88,7 @@
   &Wrapper {
     display: flex;
     margin-top: 2rem;
-    border-bottom: 1px solid utilities.$color-background-border;
+    border-top: 1px solid utilities.$color-background-border;
```

Comment:
Pourquoi le bottom a été été changé en top ? 
Du coup ça m'a intrigué, et j'suis allée voir en review app, l'affiche de la ligne de séparation est collé au bouton, et en tablette/mobile la ligne de séparation du bas me semble très proche du widget (après peut être que c'était demandé par la maquette ?)

by the way, boy scout rule, déclarer en variable comme dit le pending ADR :p

Reply:
Oula oui je me suis trompé de fichier merci 🥶